dev/Scroller/Scroller.cpp

```diff
@@ -6506,6 +6555,7 @@ winrt::hstring Scroller::DependencyPropertyToString(const winrt::IDependencyProp
     {
         return L"VerticalScrollMode";
     }
+#ifdef USE_SCROLLMODE_AUTO
```

Comment:
>USE_SCROLLMODE_AUTO [](start = 7, length = 19)

When do we expect this to be true?

Reply:
If we decide to resurrect that feature. For the time being, I would prefer to keep this code checked in, until it becomes clearer that we won't add back the Auto setting. 
We can easily remove the dead code in the future, as a standalone check-in.